spec/locales-currencies-tz.html

```diff
@@ -40,18 +40,34 @@ <h1>Unicode Locale Extension Sequences</h1>
       <h1>IsStructurallyValidLanguageTag ( _locale_ )</h1>
 
       <p>
-        The IsStructurallyValidLanguageTag abstract operation verifies that the _locale_ argument (which must be a String value)
+        The IsStructurallyValidLanguageTag abstract operation determines whether the _locale_ argument (which must be a String value) is a language tag recognized by this specification.  (It does not consider whether the language tag conveys any meaningful semantics.  It also does not differentiate between aliased subtags and their preferred replacement subtags.)
+      </p>
+
+      <p>
+        IsStructurallyValidLanguageTag returns *true* if all of the following conditions hold, *false* otherwise:
       </p>
 
       <ul>
-        <li>represents a well-formed "Unicode BCP 47 locale identifier" as specified in <a href="https://unicode.org/reports/tr35/#Unicode_locale_identifier">Unicode Technical Standard 35 section 3.2</a>,</li>
-        <li>does not include duplicate variant subtags, and</li>
-        <li>does not include duplicate singleton subtags.</li>
+        <li>_locale_ can be generated from the EBNF grammar for `unicode_locale_id` in <a href="https://unicode.org/reports/tr35/#Unicode_locale_identifier">Unicode Technical Standard #35 LDML § 3.2 Unicode Locale Identifier</a>;</li>
+        <li>_locale_ does not use any of the backwards compatibility syntax described in <a href="https://unicode.org/reports/tr35/#BCP_47_Conformance">Unicode Technical Standard #35 LDML § 3.3 BCP 47 Conformance</a>;</li>
+        <li>the `unicode_language_id` within _locale_ does not contain ASCII case-insensitively equivalent `unicode_variant_subtag` subtags; and</li>
```

Comment:
This line should be changed so reader won't misread it as there should be no variant tag at all.

Reply:
I keep staring at this and can't see this meaning in it...but also I wrote it.  ^_^  I updated this with different wording that puts "duplicate" closer to the start; let met know if there's still a problem with that.